rosdep/ruby.yaml

```diff
@@ -77,7 +85,9 @@ ruby:
 ruby-sass:
   debian: [ruby-sass]
   fedora: [rubygem-sass]
+  gentoo: [dev-ruby/sass]
   ubuntu: [ruby-sass]
 ruby1.9.3:
+  gentoo: []
```

Comment:
It's upstream deprecated and no more available on gentoo. I don't know any package that uses it.